test/integration/run-command/run_cursor_command.test.ts

```diff
@@ -0,0 +1,50 @@
+import { expect } from 'chai';
+
+import { Db, MongoClient } from '../../mongodb';
+
+describe('runCursorCommand API', () => {
+  let client: MongoClient;
+  let db: Db;
+
+  beforeEach(async function () {
+    client = this.configuration.newClient({}, { monitorCommands: true });
+    db = client.db();
+    await db.dropDatabase().catch(() => null);
+    await db
+      .collection<{ _id: number }>('collection')
+      .insertMany([{ _id: 0 }, { _id: 1 }, { _id: 2 }]);
+  });
+
+  afterEach(async function () {
+    await client.close();
+  });
+
+  it('returns each document only once across multiple iterators', async () => {
```

Comment:
Should this test be on `AbstractCursor` instead of `runCommandCursor`, since the behavior comes from the abstract class?  This test is not specific to our RunCommandCursor.

Why was it added now?